Adds missing sensors which should fix #240

JohNan opened the pull request:
* Adds missing sensors pointed out in #240 
* Remove dead code

tubalainen commented:
Tested it, added the new files, removed the component, restarted hass and added back the component again, - all ok.

But, should we make all attributes (sensors etc) checked by default in the config process? Only diesel cars would have "AdBlue" for instance? Huhuhm, might be a question down the road.

tubalainen commented:
@robinostlund please approve and push a new version. Thanks :)

JohNan commented:
> Tested it, added the new files, removed the component, restarted hass and added back the component again, - all ok.
> 
> But, should we make all attributes (sensors etc) checked by default in the config process? Only diesel cars would have "AdBlue" for instance? Huhuhm, might be a question down the road.

Since we are logging in during the config flow we should be able to filter the list of selectable instruments based on availability for the selected car.

we could save the instruments here: https://github.com/robinostlund/homeassistant-volkswagencarnet/blob/master/custom_components/volkswagencarnet/config_flow.py#L152 and then use that list to populate the dropdown.

tubalainen commented:
> > Tested it, added the new files, removed the component, restarted hass and added back the component again, - all ok.
> > But, should we make all attributes (sensors etc) checked by default in the config process? Only diesel cars would have "AdBlue" for instance? Huhuhm, might be a question down the road.
> 
> Since we are logging in during the config flow we should be able to filter the list of selectable instruments based on availability for the selected car.
> 
> we could save the instruments here: https://github.com/robinostlund/homeassistant-volkswagencarnet/blob/master/custom_components/volkswagencarnet/config_flow.py#L152 and then use that list to populate the dropdown.

Oh yeyeeye....

Huhum, it would be VERY neat if we could parse the first response from We Connect after adding the VIN and only add the supported sensors/switches for the car at hand.
